packages/manager/src/services/APIKeyService.ts

```diff
@@ -33,12 +33,14 @@ export const createMultiAPIKeys = async (environments: IEnvironment[], payload:
     if (envKey) {
       apiKey.label = envKey.label;
       apiKey.expiredDate = envKey.expiredDate;
-      apiKey.environments?.push({
-        name: env.name,
-        key: envKey.key,
-        shortKey: envKey.shortKey,
-        createdAt: envKey.createdAt,
-      });
+      if (!!apiKey.environments) {
```

Comment:
Isn't this the same as:
```suggestion
      if (apiKey.environments) {
```

Reply:
Yes, and since environments is an array (I think?), `.length` needs to be checked, because empty arrays coerce to `true`.  See:

```js
let x = [];
if (x) console.log("x is true")
>>> x is true
```